include/revng/StackAnalysis/SegregateDirectStackAccesses.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "revng/BasicAnalyses/GeneratedCodeBasicInfo.h"
+#include "revng/Support/revng.h"
+
+class SegregateDirectStackAccessesPass
+  : public llvm::PassInfoMixin<SegregateDirectStackAccessesPass> {
+  llvm::LLVMContext *Context = nullptr;
+  GeneratedCodeBasicInfo *GCBI = nullptr;
+  std::vector<llvm::Instruction *> DirectStackAccesses;
+  std::vector<llvm::Instruction *> NotDirectStackAccesses;
+
+public:
+  SegregateDirectStackAccessesPass() = default;
```

Comment:
Drop

Reply:
The default constructor is available by default here, so no point. I don't about other situations.
Just test it, if it works without it, drop it.

Reply:
Got it.